addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -411,10 +411,10 @@
                         ":has(label:contains('State'))" +
                         ":has(select[required]:hidden)" +
                         ":has(.s_website_form_select_item:contains('Belgium'))" +
-                        ":has(.s_website_form_select_item:contains('France'))" +
-                        ":has(.s_website_form_select_item:contains('Canada'))" +
-                        ":has(.s_website_form_select_item:contains('44 - UK'))" +
-                        ":not(:has(.s_website_form_select_item:contains('Germany')))",
+                        ":has(option:contains('France'))" +
+                        ":has(option:contains('Canada'))" +
+                        ":has(option:contains('44 - UK'))" +
+                        ":not(:has(option:contains('Germany')))",
```

Comment:
It may be worth improving the test to check both ways to display such a field, instead of just switching to the other way.